src/util/color.hpp

```diff
@@ -0,0 +1,14 @@
+#include "backend/looped/looped.hpp"
+#include "natives.hpp"
+#include "script.hpp"
+
+namespace big
+{
+	inline class rgb
+	{
+	public:
+		float red   = 255;
+		float green = 0;
+		float blue  = 0;
+	} rgb{};
+}
```

Comment:
Why are you saving data under util?

Reply:
couldnt think of a different place since its being accessed by multiple files

Reply:
`core/data/color.hpp`?